content/api/api-alerts.md

```diff
@@ -0,0 +1,180 @@
+---
+title: "Alerts API"
+draft: false
+categories:
+tags: ["#api", "#alerts"]
+author: Lawrence Lane
+alwaysopen: false
+pre: ""
+---
+
+[Alerts][1] group anomalous behavior and are informed by multi-conditional policies you select or build in CloudWisdom. Through the Alerts API, you can:
+
+- Get a list of alerts, filterable by properties
+- Close an alert via ID
+- Get an alert via ID
+
+## Parameters
```

Comment:
Do you think it's worth breaking this table up and adding the relevant params to each endpoint as we have in other docs? It gives the impression each endpoint can accept all the params.

Reply:
I was torn on which way to do it. I'm happy to break them back out or just have one static reference table.

Reply:
We agreed to break them out in a meeting.